cmd/hemictl/README.md

```diff
@@ -1,36 +1,46 @@
-## hemictl
+![image](https://github.com/user-attachments/assets/0d0ea1df-5f9a-42b8-a42d-15fb814e5ac9)## hemictl
```

Comment:
I don't think this image is supposed to be here. Could you remove it please?
![image](https://github.com/user-attachments/assets/591f2ab1-599b-4c72-930c-aa36b1301f73)